src/MaterialUI.re

```diff
@@ -1309,6 +1311,42 @@ module Select = {
     );
 };
 
+module Slide = {
+  [@bs.module "material-ui/transitions/Slide"] external slide : ReasonReact.reactClass = "default";
+  module Direction = {
+    type t =
+      | Up
+      | Down
+      | Left
+      | Right;
+    let to_string =
+      fun
+      | Up => "up"
+      | Down => "down"
+      | Left => "left"
+      | Right => "right";
+  };
+  let make =
+      (
+        ~in_: option(bool)=?,
+        ~direction_: option(Direction.t)=?,
+        ~timeout: option({. "enter": float, "exit": float})=?,
+        children
+      ) =>
```

Comment:
There are also some events missign which have been omitted from the docs.

  onEnter?: TransitionCallback, 
  onEntering?: TransitionCallback,
  onEntered?: TransitionCallback,
  onExit?: TransitionCallback,
  onExiting?: TransitionCallback,
  onExited?: TransitionCallback,

Reply:
Should also include the properties of the Transition class:

  appear?: boolean;
  unmountOnExit?: boolean;